src/main/resources/dashboard/user.hbs

```diff
@@ -1,22 +1,35 @@
 {{#partial "content"}}
 
 <p>
-  You may reset your password to be used in
-  authentication through package managers, like Maven:
+  To start working with Artipie using package managers you have to obtain
+  GitHub <a href="https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token">personal access token</a> to use it as a password. You username will be <code>github.com/{{user}}</code>.
```

Comment:
@g4s8 hm, I think we should simplify this sentence:
Obtain GitHub personal access token and use it as a password in authentication through package managers, your username is github.com/{{user}}.